src/components/logoCard/LogoCard.tsx

```diff
@@ -0,0 +1,85 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import * as  s from 'underscore.string';
+import { Badge, IBadgeProps } from '../badge/Badge';
+import { CornerRibbon, DEFAULT_CORNER_RIBBON_CONTAINER_CLASSNAME, ICornerRibbonProps } from '../cornerRibbon/CornerRibbon';
+import { Svg } from '../svg/Svg';
+
+export const DEFAULT_LOGO_CARD_CLASSNAME: string = 'logo-card';
+export const DEFAULT_LOGO_ICON: string = 'source-custom';
+export const DEFAULT_LOGO_ICON_CLASSNAME: string = 'icon';
+export const DEFAULT_LOGO_ICON_SIZE: string = 'mod-4x';
+export const DEFAULT_DISABLED_RIBBON_LABEL: string = 'Unavailable';
+
+export interface ILogoCardProps {
```

Comment:
Je sais pas si ReactVapor c'est un exception mais dans le styleguide on dit de pas préfixer les interfaces avec I

Reply:
Ouin, on a ça un peu partout dans r-vapor, je suis d'avis aussi qu'on devrait pas mettre de `I` though.

Reply:
It dates back to when we add `pretty-typescript`. It would force us to add `I`s to the interface. Now that we don't have it anymore, we could remove them, but it'd be a breaking change. We could just not add it anymore though. But it wouldn't be great for uniformity.